connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockCommandRequestTranslator.java

```diff
@@ -45,7 +46,11 @@ public void translate(CommandRequestPacket packet, GeyserSession session) {
         if (session.getConnector().getPlatformType() == PlatformType.STANDALONE && command.startsWith("geyser ") && commandMap.getCommands().containsKey(command.split(" ")[1])) {
             commandMap.runCommand(session, command);
         } else {
-            ClientChatPacket chatPacket = new ClientChatPacket(packet.getCommand());
+            String message = packet.getCommand().trim();
+
+            if (MessageUtils.isTooLong(message, session)) { return; }
```

Comment:
Should be:
```java
if (MessageUtils.isTooLong(message, session)) {
    return;
}
```
Please try and avoid returning on the same line the if statement is on.